test/e2e/failover_test.go

```diff
@@ -141,6 +142,106 @@ var _ = framework.SerialDescribe("failover testing", func() {
 			})
 		})
 	})
+
+	ginkgo.Context("Taint cluster testing", func() {
+		var policyNamespace, policyName string
+		var deploymentNamespace, deploymentName string
+		var deployment *appsv1.Deployment
+		var taint corev1.Taint
+		var maxGroups, minGroups, numOfFailedClusters int
+		var policy *policyv1alpha1.PropagationPolicy
+		maxGroups = 1
```

Comment:
maxGroups, minGroups, numOfFailedClusters := 1,1,1

Reply:
/cc @XiShanYongYe-Chang. I follow the former code style. To be honest, I do not know which style is better.

Reply:
It's just a code style, any writing is OK. Keep the overall consistency will be nice.